lib/types/db.ts

```diff
@@ -25,6 +26,17 @@ export type CurrencyAttributes = CurrencyFactory & {
 
 export type CurrencyInstance = CurrencyAttributes & Sequelize.Instance<CurrencyAttributes>;
 
+// SwapDeals
+export type SwapDealFactory = SwapDeal;
+
+export type SwapDealAttributes = SwapDealFactory & {
```

Comment:
I think the attributes is simply equal to the factory type in this case, those two time fields can already be undefined.